bin/public/installHaproxy.py

```diff
@@ -139,7 +146,8 @@ def _configure_haproxy():
 
     scopen.scOpen(HAPROXY_CONF).replace("${ENV_IP}", get_ip_address('eth1'))
     _configure_haproxy_state()
-    _configure_sps_password()
+    if haproxy_env() == "farepayment":
```

Comment:
This type of internal code can't be in the generic installer. We need to find a way to move this away from the public codebase...